meja/envi.ml

```diff
@@ -117,4 +117,4 @@ let get_name name env =
   match List.find_map ~f:(Scope.get_name name) env.scope_stack with
   | Some (In_progress typ) -> (typ, env)
   | Some (Final typ) -> Type.import typ env
-  | None -> failwith "Could not find name."
+  | None -> raise (Not_found_s (Atom "Could not find name in the environment"))
```

Comment:
Maybe `sprintf` in the name?

Reply:
It gets immediately caught in `typecheck.ml`. I only used `Not_found_s` because `Not_found` yells at me that it's deprecated.